tensorboard/uploader/proto/server_info.proto

```diff
@@ -20,6 +20,16 @@ message ServerInfoResponse {
   ApiServer api_server = 2;
   // How to generate URLs to experiment pages.
   ExperimentUrlFormat url_format = 3;
+  // For which plugins should we upload data? (Even if the uploader is
+  // structurally capable of uploading data from many plugins, we only actually
+  // upload data that can be currently displayed in TensorBoard.dev. Otherwise,
+  // users may be surprised to see that experiments that they uploaded a while
+  // ago and have since shared or published now have extra information that
+  // they didn't realize had been uploaded.)
+  //
+  // If this field is omitted, the behavior of which data to send is at the
```

Comment:
Maybe reword this comment a bit: which data to send is still at the discretion of the client, within the constraints given by this field.

Reply:
Good point. Done.